litmus/director/tcid-iuoc02-control-plane-upgrade-operator-yml/test.yml

```diff
@@ -0,0 +1,106 @@
+---
+- hosts: localhost
+  connection: local
+
+  vars_files:
+    - test_vars.yml
+
+  tasks:
+
+    - block:
+  
+        ## Generating the testname for deployment
+        - include_tasks: /ansible-utils/create_testname.yml
+
+        ## RECORD START-OF-TEST IN LITMUS RESULT CR
+        - include_tasks: /ansible-utils/update_litmus_result_resource.yml
+          vars:
+            status: 'SOT'
+          
+        - set_fact:
+            director_url : "http://{{ director_ip }}:30380"
```

Comment:
Why is the `director_url` being set here if we are already passing it as a configmap to the job?

Reply:
we are passing `director_ip` in job but here we are adding port into the `director_ip`

Reply:
@amitbhatt818 the director URL configmap is being created using both IP and port --
![image](https://user-images.githubusercontent.com/40303207/79101978-682c1d80-7d87-11ea-8a86-c8fc5a9be172.png)